### PR TITLE
Merge from user-input-branch development branch.

### PR DESCRIPTION
**Pull request by qu-gg**
- Add user input to Gun generation
- Add generate_gun_cmd to generate Gun Card PDFs with user input via argparse in CMD
- Add template for the Gun Card and an example output with random generation

### CODE REVIEW

File: classes/GunImage.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            "https://global-uploads.webflow.com/5ff36780a1084987868ce198/618fd0cdd6736d4a1bbf5fbe_9-Volt%20(SMG-BL3).png",
+            "https://global-uploads.webflow.com/5ff36780a1084987868ce198/618fdc07ad989ce7693372d0_Narp%20(Sniper-BL3).png"
+        ]
+        url = random.sample(url, 1)[0]
+
+        response = requests.get(url, stream=True)
+        img = Image.open(response.raw)
+        img.save('temporary_gun_image.png')
+
+    def manu_conversion(self, guild_name):
```
Comment (Milkshak3s) [inline classes/GunImage.py:111]: Using a dict for this is more efficient than direct comparison:

```
manu_map = {
  "alas!": "Atlas"
}

return manu_map.get(guild_name, None)
```

File: generate_gun_cmd.py [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+import pdfrw
+import locale
+import argparse
+import subprocess
+
+from pdfrw import PdfReader
+from classes.Gun import Gun
+from classes.GunImage import GunImage
+
+# KEY Names for PDF
```
Comment (Milkshak3s) [inline generate_gun_cmd.py:21]: Wow you went hard on PDFs here. I am going to skip this for review

File: resources/images/gun_data.py [hunk 3]
```diff
@@ -1,52 +0,0 @@
-"""
-@file gun_data.py
```
Comment (Milkshak3s) [inline resources/images/gun_data.py:2]: Hm I added a method in my most recent PR, prepare for uncomfortable merge issues

**Review by Milkshak3s — CHANGES_REQUESTED** [review]

File: classes/GunImage.py (hunk 1 above)
Comment (qu-gg) [reply classes/GunImage.py:111]: Ahhh good catch. Fever dream late night pumping lmao

File: generate_gun_cmd.py (hunk 2 above)
Comment (qu-gg) [reply generate_gun_cmd.py:21]: Misery

File: resources/images/gun_data.py (hunk 3 above)
Comment (qu-gg) [reply resources/images/gun_data.py:2]: Yee I'll work it out.

Comment (qu-gg) [reply resources/images/gun_data.py:2]: Actually should be fine. I shifted over logic to a GunImage class under classes/, so can accept your changes.
Then I'll merge them together for consistency.

**Pull request merged**